docs/faq.md

```diff
@@ -6,6 +6,14 @@ Action Scheduler is designed to be used and released in plugins. It avoids redec
 
 To use it in your plugin (or theme), simply require the `action-scheduler/action-scheduler.php` file. Action Scheduler will take care of the rest. __Note:__ Action Scheduler is only loaded from a theme if it is not included in any active plugins.
 
+### How to disable Action Scheduler?
+
+Disabled it is helpful for automated testing and it is enough to add a constant in your `wp-config.php` as:
+
+```
+define( 'AS_DISABLE_QUEUE_RUNNERS', true );
+```
```

Comment:
✍🏽 Once we're happy with the change (and can be confident about the version it will become available) we could consider adding a further note here, such as:

```md
This is available from Action Scheduler x.y.z onwards.
```